### ISSUE TITLE
failed to resolve: use of undeclared crate or module `tracing`

### ISSUE THREAD

**Issue by frederikhors** [tokio-rs/axum#2932]
I'm getting this error:

```
error[E0433]: failed to resolve: use of undeclared crate or module `tracing`
   --> C:\Users\fred\.cargo\registry\src\index.crates.io-6f17d22bba15001f\axum-extra-0.9.4\src\typed_header.rs:161:9
    |
161 |         axum_core::__log_rejection!(rejection_type = Self, body_text = body, status = status,);
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `tracing`
    |
    = note: this error originates in the macro `axum_core::__log_rejection` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Why?

**Comment by jplatte** [comment]
Well, that's a bug, clearly. Could you share your (axum-related) dependencies from `Cargo.toml`?

**Comment by jplatte** [comment]
Actually I already know what's up. Will have a fix ready shortly. What you can do for now is making sure that the `tracing` feature of `axum-extra` is enabled (assuming you want to have rejections logged) or making sure that the `tracing` feature is disabled for all the axum crates.

**Comment by jplatte** [comment]
Fix has been released in axum-core v0.4.5!
